variables.tf

```
@@ -26,6 +26,12 @@ variable "location" {
   default     = ""
 }
 
+variable "storage_class" {
+  description = "The storage class of the GCS bucket. Defaults to 'STANDARD' if nothing is specified here. Available options include 'STANDARD', 'MULTI_REGIONAL', 'REGIONAL', 'NEARLINE', 'COLDLINE', 'ARCHIVE'. See https://cloud.google.com/storage/docs/storage-classes."
```

```suggestion
  description = "The storage class of the GCS bucket. Defaults to 'STANDARD' if nothing is specified here. Available options include 'STANDARD', 'NEARLINE', 'COLDLINE', 'ARCHIVE'. See https://cloud.google.com/storage/docs/storage-classes."
```

Regional, Multi-Regional are not storage "CLASS". Ref: https://cloud.google.com/storage/docs/storage-classes

---

It's in additional classes https://cloud.google.com/storage/docs/storage-classes#legacy. Seems like it is some legacy class and we should be using STANDARD instead anyways. Will amend.